### ISSUE TITLE
Support for blender >2.8?

### ISSUE THREAD

**Issue by vdmkenny** [Kewka/dffconvert#1]
Can this plugin be updated for recent plugin versions? Is there a reason it can't?

**Comment by kewka** [comment]
Yes, this plugin can be updated. I don't have time for this update :smiley_cat:. 2.80 release contains breaking changes ([release notes](https://wiki.blender.org/wiki/Reference/Release_Notes/2.80/Python_API)).  I think you can override the behavior depending on the [bpy.app.version](https://docs.blender.org/api/current/bpy.app.html#bpy.app.version)
